hey @ferdymercury - just testing this out! I have an up to date pydicom and just installed tix-dev, I'm still hitting an issue:

```python
sudo apt install tix-dev
sudo pip install -U pydicom
python3 dicomtree.py file.dcm
=========================================
Show a dicom file using hierarchical tree
=========================================
Show a dicom file using a hierarchical tree in a graphical window.

Traceback (most recent call last):
  File "dicomtree.py", line 84, in <module>
    root = tkinter_tix.Tk()
  File "/home/vanessa/anaconda3/lib/python3.7/tkinter/tix.py", line 214, in __init__
    self.tk.eval('package require Tix')
_tkinter.TclError: can't find package Tix
```
I have Ubuntu 18.04, and I'm guessing that other users might hit this as well. Do you have any suggestions? I'd be glad to test, if I can get the script to work. If there is consistent issue, you could provide a Dockerfile alongside the image that builds the dependencies as needed.

--- a/plotting-visualization/README.md
+++ b/plotting-visualization/README.md
@@ -3,5 +3,6 @@
 This folder is intended for examples that are scoped outside of the "medical image viewer" use case. This might include statistical plots, charts for data analysis, or interactive renderings of data for the web.
 
  - [dcm_qt_tree.py](dcm_qt_tree.py): View DICOM files in a tree using Qt and PySide.
+ - [dicomtree](dicomtree.py): View DICOM files and tags in a tree using Tkinter.
  - [matplotlib-dicom](matplotlib-dicom): The simplest action to read in a dicom file with pydicom, and plot with matplotlib.
 

--- a/plotting-visualization/dicomtree.py
+++ b/plotting-visualization/dicomtree.py
@@ -3,6 +3,8 @@
 sudo apt install tix-dev
 sudo pip install -U pydicom
 python3 dicomtree.py file.dcm
+Or in file browser, right click on file, open with custom command:
+python3 dicomtree.py
 =========================================
 Show a dicom file using hierarchical tree
 =========================================
@@ -82,6 +84,7 @@ if __name__ == '__main__':
         print(usage)
         sys.exit(-1)
     root = tkinter_tix.Tk()
+    root.title("DICOM tree viewer - "+sys.argv[1])
     root.geometry("{0:d}x{1:d}+{2:d}+{3:d}".format(1200, 900, 0, 0))
 
     RunTree(root, sys.argv[1])

